client/hack/README.md

```diff
@@ -26,7 +26,7 @@ Make sure to run this script after making changes to /client/apis/volumesnapshot
     ```
 * Checkout latest release version
     ```bash
-    git checkout v0.25.2
+    git checkout v0.26.0
```

Comment:
Can you use 0.26.1? https://github.com/kubernetes-sigs/controller-tools/blob/v0.11.3/go.mod#L16

Reply:
Updated